client/layout/logged-out.jsx

```diff
@@ -32,6 +33,7 @@ module.exports = React.createClass( {
 				<Masterbar />
 				<div id="content" className="wp-content">
 					<NoticesList id="notices" notices={ notices.list } />
+					<GlobalNotices id="notices" />
```

Comment:
Can those two components be both rendered at the same time? If that happens, won’t the two HTML elements have the same `id`?